client/src/components/LevelLayout.tsx

```diff
@@ -33,6 +34,9 @@ const LevelLayout: React.FC<Props> = ({
           {headingText}
         </Heading>
         <Spacer />
+        <Heading as="h1" fontSize="lg" mr={4} mt={2}>
```

Comment:
nit: semantically, I don't think specifying `as="h1"` here is best practice. The timer value isn't the heading/title of the page. An alternative could be to use `<Text />` with `fontWeight="bold"`.

Reply:
Awesome, I changed this to Chakra UI Text